_docs/migration.md

```diff
@@ -1,115 +1,152 @@
+- They have been replaced by `.hidden-xs-up` `.hidden-xs-down` `.hidden-sm-up` `.hidden-sm-down` `.hidden-md-up` `.hidden-md-down` `.hidden-lg-up` `.hidden-lg-down`.
+- The `.hidden-*-up` classes hide the element when the viewport is at the given breakpoint or larger (e.g. `.hidden-md-up` hides an element on medium, large, and extra-large devices).
+- The `.hidden-*-down` classes hide the element when the viewport is at the given breakpoint or smaller (e.g. `.hidden-md-down` hides an element on extra-small, small, and medium devices).
+
+Rather than using explicit `.visible-*` classes, you make an element visible by simply not hiding it at that screen size. You can combine one `.hidden-*-up` class with one `.hidden-*-down` class to show an element only on a given interval of screen sizes (e.g. `.hidden-sm-down.hidden-xl-up` shows the element only on medium and large devices).
+
+Note that the changes to the grid breakpoints in Athena means that you'll need to go one breakpoint larger to achieve the same results (e.g. `.hidden-md` is more similar to `.hidden-lg-down` than to `.hidden-md-down`). The new responsive utility classes don't attempt to accommodate less common cases where an element's visibility can't be expressed as a single contiguous range of viewport sizes; you will instead need to use custom CSS in such cases.
+
+### Components
+
+- **Dropped panels, thumbnails, and wells** for a new all-encompassing component, [cards]({{ site.baseurl }}{% link components/card.md %}).
+- **Dropped the Glyphicons icon font.** If you need icons, we recommend including [Font Awesome](http://fontawesome.io/) or another icon library in your project as needed.
+- **Dropped the Affix jQuery plugin.** Use the [`.sticky-top` utility class]({{ site.baseurl }}{% link utilities/position.md %}#sticky-top) instead.
+  - If you were using Affix to apply additional, non-`position` styles, `.sticky-top` might not support your use case. One option for such uses is the third-party [ScrollPos-Styler](https://github.com/acch/scrollpos-styler) library.
+- **Dropped the pager component** as it was essentially slightly customized buttons.
+- **Refactored nearly all components** to use more un-nested classes instead of children selectors.
+
+
+---
+
+
+## By component
+
+This list highlights key changes by component between Bootstrap v3.x.x and Athena.
+
+### Breadcrumbs
+
+- An explicit class, `.breadcrumb-item`, is now required on the descendants of `.breadcrumb`s.
 
 ### Buttons
 
 - Dropped the `.btn-xs` class entirely.
-- The [stateful button](https://getbootstrap.com/javascript/#buttons-methods) feature of the `button.js` jQuery plugin has been dropped. This includes the `$().button(string)` and `$().button('reset')` methods. We advise using a tiny bit of custom JavaScript instead, which will have the benefit of behaving exactly the way you want it to.
```

Comment:
The 'stateful button' link redirects from the given url to [https://getbootstrap.com/docs/3.3/javascript/](url).